t/CRMS.t

```diff
@@ -18,6 +18,27 @@ subtest '#Version' => sub {
   ok($crms->Version);
 };
 
+subtest '#extract_env_email' => sub {
```

Comment:
There could be a test for "email is empty string", just to make obvious what happens.
The tests read well and test well.

```
$ENV{email} = '';
is_deeply($crms->extract_env_email, [], 'empty string -> []');
# or something
```